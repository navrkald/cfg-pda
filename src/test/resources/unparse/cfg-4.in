S => 0 | 1 | A | e
A => 0A0 | e