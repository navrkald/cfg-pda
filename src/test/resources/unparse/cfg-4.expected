S => 0
S => 1 
S => A 
S => e
A => 0A0 
A => e
